src/xdocs/releasenotes.xml

```diff
@@ -5847,7 +5847,8 @@ used in the MT mode.
           move non-compilable input to resources-noncompilable. Author: Vladislav Lisetskii
         </li>
         <li>
-          Minor update in <a href="config_coding.html#FallThrough">FallThroughCheck</a> xdoc and javadoc. Author: Vladislav Lisetskii
```

Comment:
@romani I thought release notes was not subject to this validation or am I wrong?
I know we are still fixing the utility to be better on line lengths.

Reply:
We can fix links, but it is not good to enforce this in build, nobody review release notes diff, so validation of length is not required. Release notes generation tool should generate content in format we want.

Reply:
@pbludov Can you confirm if this validation is being enforced on this file?

Reply:
done. Added suppression for `releasenotes.xml`, the file itself reverted.